### ISSUE TITLE
sorry I thought I was replying to a message please delete this

### ISSUE THREAD

**Issue by evoc2911** [moraroy/NonSteamLaunchers-On-Steam-Deck#382]
> @evoc2911 OK, open up the new desktop file and hit Stop Game Scanner, when it asks if you want to run it again hit NO. The new NinSteamLaunchers shortcut that was created for you is just a placeholder, simply Hide it in your library, right click the shortcut and go to manage and click on Hide this Game. It's for protontricks. Now, exit steam fully basically restart steam. And let me know if your launchers work again, if they don't, then I would check and see what's going on in your compatadata folder located at /home/deck/.local/share/steam/steamapps/compatdata/ let me know what folders are in there.

Hi man, appreciated a lot your help. I did as you said but the launcher still won't launch. here's my compdata picture, once opened the NSL folder:
![steammapp](https://github.com/user-attachments/assets/b252fa33-de71-4960-a0a5-bee427f60e97)

_Originally posted by @evoc2911 in https://github.com/moraroy/NonSteamLaunchers-On-Steam-Deck/issues/264#issuecomment-2265436946_

**Comment by evoc2911** [comment]
**sorry I thought I was replying to a message please delete this**

**Comment by moraroy** [comment]
- @evoc2911 the directory before that one. The compatdata

**Comment by evoc2911** [comment]
Hi moraroy, I just managed to reinstall battle.net. Updated protonGE on NSL - battle.net launched and now I'm updating the game ( WoW ).
Here's how my compdata folder looks now ( there was previousòy a NSL folfer, but before reinstalling battle.net I cliccked on "start from scratch" )
![compdata 8 Aug](https://github.com/user-attachments/assets/abc37852-fe0a-4d53-bf35-ccd8914f01cb)

Thanks again for helping me!
